src/module/utils/TwodsixRollSettings.ts

```diff
@@ -0,0 +1,99 @@
+import {TWODSIX} from "../config";
+import type TwodsixItem from "../entities/TwodsixItem";
+import { getKeyByValue } from "./sheetUtils";
+
+
+export class TwodsixRollSettings {
```

Comment:
Roll settings. It has a set of default settings which it will apply if it doesn't receive any specific settings. Can also ask for specifics through the dialog